Source/WarpX.cpp

```diff
@@ -889,20 +893,23 @@ WarpX::AllocLevelMFs (int lev, const BoxArray& ba, const DistributionMapping& dm
     if (fft_hybrid_mpi_decomposition == false){
         // Allocate and initialize the spectral solver
         std::array<Real,3> dx = CellSize(lev);
-#if (AMREX_SPACEDIM == 3)
+    #if (AMREX_SPACEDIM == 3)
```

Comment:
I would keep preprocessor things left-aligned since this could otherwise suggest that the syntax around it would matter (it does not and is bare-handed text snipping before parsing).